Distance_2/test/Distance_2/test_distance_2.cpp

```diff
@@ -296,7 +290,7 @@ struct Test {
 
 int main()
 {
-	Test< CGAL::Cartesian<double>   >().run();
+	Test< CGAL::Simple_cartesian<double>   >().run();
 	Test< CGAL::Homogeneous<double> >().run();
```

Comment:
And why not `Simple_homogeneous` as well?

Reply:
In fact...